pulp_ansible/app/tasks/collectionversion_index.py

```diff
@@ -93,17 +129,14 @@ def update_index(distribution=None, repository=None, repository_version=None, is
         if CVIndex.objects.filter(repository_version=repository_version).exists():
             return
 
+    # What has changed between this version and the last?
+    changed_collections = compute_repository_changes(repository_version)
+
     # get all CVs in this repository version
-    cvs = repository_version.content.filter(pulp_type="ansible.collection_version").values_list(
+    cvs_pk = repository_version.content.filter(pulp_type="ansible.collection_version").values_list(
```

Comment:
I'm kind of used to this convention. (Feel free to ignore)
```suggestion
    cv_pks = repository_version.content.filter(pulp_type="ansible.collection_version").values_list(
```